src/main/resources/org/sagebionetworks/web/client/widget/sharing/AclAddPeoplePanel.ui.xml

```diff
@@ -3,33 +3,28 @@
 	xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
 	xmlns:bh="urn:import:org.gwtbootstrap3.client.ui.html"
 	xmlns:g="urn:import:com.google.gwt.user.client.ui">
-
-	<!-- <g:HTMLPanel ui:field="mainContainer" addStyleNames="panel panel-default"> -->
 	<bh:Div>
-		<b:Heading size="H5" text="Add More People"
-			addStyleNames="margin-top-40" />
-		<g:Label
-			text="Search for a Synapse user or team below.  Click on the user or team name, then adjust their access level above before clicking Save."
-			addStyleNames="margin-bottom-10" />
-		<b:Panel>
-			<b:PanelBody>
-				<!-- <b:Container> -->
+		<b:Heading size="H4" text="Add More People"
```

Comment:
There are some other UI changes made here based on the mockups.  Removed the Panel, and italicized some text.